src/components/BookList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { v4 as uuid4 } from 'uuid';
+import ContentBook from './ContentBook';
+import BookForm from './BookForm';
+
+const BookList = () => {
+  const books = [
+    {
+      title: 'the gods are not to blame',
+      author: 'Ola Rotimi',
+      completed: true,
+      id: uuid4(),
+    },
+    {
+      title: 'The marriage of anansewa',
+      author: 'ama ataa aidoo',
+      completed: true,
+      id: uuid4(),
+    },
+    {
+      title: 'ancestral sacrifice',
+      author: 'akosombo nyantakyi',
+      completed: true,
+      id: uuid4(),
+    },
+  ];
```

Comment:
- [ ] Kindly remove this list of books you have in this file as you don't need it. You're expected to make use of `useSelector` from `react-redux` to get all books from your `booksSlice` and populate it here nicely as you have done.

Reply:
removed as suggested